### PR TITLE
[Components with more than 1 service] LPS-188721 Refactor ModelListener components part II

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://liferay.atlassian.net/browse/LPS-188721

@dantewang I decided to use a *Util class instead of a Helper because I found out a way to use the *Util class in the integration test, so I don't have to create another component.

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:search

**Comment by lilylcl** [comment]
ci:test:stable

**Comment by lilylcl** [comment]
The failures of all test suites are not related to the current PR. @slnn Could you help to add QA Passed label? Thanks.

File: modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/internal/model/listener/test/CompanyModelListenerTest.java [hunk 1]
```diff
@@ -43,8 +46,13 @@ public class CompanyModelListenerTest {
 
 	@Test
 	public void testSXPElements() throws Exception {
+		Bundle bundle = FrameworkUtil.getBundle(_modelListener.getClass());
```
Comment (dantewang) [inline modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/internal/model/listener/test/CompanyModelListenerTest.java:49]: @MrJohn1911 Please rename this test and move it to proper package, others look good to me, thanks!

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
no regressions

**Comment by dantewang** [comment]
ci:test:stable

**Review by dantewang — APPROVED** [review]

**Comment by dantewang** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/137595

**Comment by dantewang** [comment]
Hi @MrJohn1911, please also see https://github.com/MrJohn1911/liferay-portal/pull/29, try also to apply the way in this one. I've closed the PR sent to BChan. Thanks!

**Pull request closed without merging**